chore: render GitKraken instructions to html

ns-rse opened the pull request:
Adding the `.html` version along with a comment in the `.Rmd` on the need to include this.

I don't know why the `<script>` sections are now included but have viewed the file locally in my browser and it looks ok.

Good sleuthing through the log history to find that I'd done this once in the past @tdjames1 :mag:

ns-rse committed:
chore: render GitKraken instructions to html

ns-rse commented:
Damn this hasn't worked, the GitKraken page isn't in the [preview](https://deploy-preview-130--peppy-otter-7a55ad.netlify.app/setup/).

I'll look at it again tomorrow (or later).

tdjames1 commented on content/setup/gitkraken/_index.en.html:
```diff
@@ -1,19 +1,365 @@
-title: Gitkraken Client
-weight: 5
----
+<!DOCTYPE html>
```
It looks like it should not be a full html page as rendered by `rmarkdown::render()` but just the html for the content section, with a yaml type preamble giving the menu item.

tdjames1 commented on content/setup/gitkraken/_index.en.html:
```diff
@@ -31,7 +377,7 @@ <h3>Integrating GitKraken Client with GitHub</h3>
 GitKraken Client. Follow <a href="https://help.gitkraken.com/gitkraken-client/github-gitkraken-client/">the
 instructions on the GitKraken site</a> to perform both steps.</p>
-<iframe src="https://help.gitkraken.com/gitkraken-desktop/github-gitkraken-desktop/" width="672" height="400px" data-external="1">
+<iframe src="https://help.gitkraken.com/gitkraken-client/github-gitkraken-client/" width="672" height="400px" data-external="1">
```
@jmafoster1 fixed this link in e8347fa but it's still wrong in the .Rmd - need to change both to the gitkraken-desktop URL.

tdjames1 committed:
Attempt to revert .html to the correct format

- Re-instate yaml header
- Re-instate script link
- Remove html except the contents
- Fix iframe links in .html and .Rmd

tdjames1 commented:
I've made an attempt to fix the html file in #131, hope it helps!

ns-rse committed:
Merge pull request #131 from RSE-Sheffield/tdj/deploy-fix-html

Attempt to revert .html to the correct format

ns-rse committed:
Merge branch 'master' into ns-rse/deploy

tdjames1 commented:
@ns-rse I think the merge from master broke things because you'd already committed the rendered html there :crying_cat_face:

ns-rse committed:
fix: Correcting messed up merge conflict

ns-rse commented:
> @ns-rse I think the merge from master broke things because you'd already committed the rendered html there 😿

Bingo! Thank you, corrected and both of the deploy previews now have the correct link included. Thanks for your patience @tdjames1
